### PR TITLE
Feature/update events

### PR DESCRIPTION
**Pull request by efratush**
Add function of event update + tests
for issue  #35

### CODE REVIEW

File: app/database/database.py [hunk 1]
```diff
@@ -5,11 +5,21 @@
 from sqlalchemy.orm import sessionmaker
 
 
-SQLALCHEMY_DATABASE_URL = os.getenv("DATABASE_CONNECTION_STRING")
+SQLALCHEMY_DATABASE_URL = "sqlite:///./calendar.db"
+# (os.getenv("DATABASE_CONNECTION_STRING"))
```
Comment (PureDreamer) [inline app/database/database.py:9]: Please remove :)

File: app/database/models.py [hunk 2]
```diff
@@ -22,8 +22,12 @@ class Event(Base):
 
     id = Column(Integer, primary_key=True, index=True)
     title = Column(String)
+    start_date = Column(DateTime)
+    end_date = Column(DateTime)
+    VC_link = Column(String)
```
Comment (PureDreamer) [inline app/database/models.py:27]: should be with lower cases

File: app/routers/event.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    return templates.TemplateResponse("eventedit.html", {"request": request})
+
+
+def get_event_by_id(db: Session, event_id: int) -> Event:
+    """Select event by id"""
+    return db.query(Event).filter(Event.id == event_id).first()
+
+
+def check_validation(start_time: datetime, end_time: datetime) -> bool:
+    """Check if the start date is smaller then the end time"""
```
Comment (PureDreamer) [inline app/routers/event.py:28]: You should use start time and not start date and also should be stated as earlier and not smaller.

File: tests/conftest.py [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+    session.close()
+    Base.metadata.drop_all(bind=engine)
+
+
+def get_test_placeholder_user():
+    return User(
+        username='fake_user',
+        email='fake@mail.fake',
+        password='123456fake',
+        full_name='FakeName'
```
Comment (PureDreamer) [inline tests/conftest.py:45]: Consider using Faker instead

**Review by PureDreamer — COMMENTED** [review]
very good piece of code!

File: app/database/database.py (hunk 1 above)
Comment (efratush) [reply app/database/database.py:9]: Right, not the updated version

File: tests/conftest.py (hunk 4 above)
Comment (efratush) [reply tests/conftest.py:45]: The realization is taken from the master code

File: app/routers/event.py (hunk 3 above)
Comment (efratush) [reply app/routers/event.py:28]: Thanks :)

**Comment by efratush** [comment]
due to some mistakes on hendling git, I am gonna rewrite the it

**Pull request closed without merging**